en/user_guide/media.rst

```diff
@@ -0,0 +1,65 @@
+Media Management
+================
+
+Open Orchestra allows to manage medias upload by users in the media library panel.
+These ones can be easily adding to different blocks or contents type later.
+Natively, images, PDF and videos are supported by Open Orchestra.
+
+Folder
+------
+
+All files are organised inside folders. It's possible to browse the folder tree to see files by clicking on the left panel.
+
+Create
+~~~~~~
+
+Creating a folder is done by clicking on the "new folder" button in the left panel.
+
+The creation form needs:
+
+* name: name of the folder
+* Websites: which websites are using the folder
+
+Remove
+~~~~~~
+
+To remove a folder, select the folder and click on the trash button on the right panel.
+
+Manage files
+------------
+
+The add button in the folder's content allows to upload files.
+
+A trash icon appears on media preview hovering to delete the media.
```

Comment:
You can add : ``The media can't be deleted if it is used in an element (block, content) which is in the final state``